docs/sources/reference/commandline/cli.md

```diff
@@ -1635,6 +1636,10 @@ timestamp, for example `2014-09-16T06:17:46.000000000Z`, to each
 log entry. To ensure that the timestamps for are aligned the
 nano-second part of the timestamp will be padded with zero when necessary.
 
+The `docker logs --since` argument shows logs of a container generated only
```

Comment:
Reading to again, perhaps make it more in line with the paragraphs above;

"The `docker logs --since` option will filter the output and only show log entries generated after the given date, specified as RFC 3339 or UNIX timestamp."

Would be nice to have some examples; can be similar to the "Show events in the past from a specified time" examples below https://docs.docker.com/reference/commandline/cli/#filtering